flake.nix

```diff
@@ -1,6 +1,7 @@
 {
   description = "Enarx Cod(e) Ex(amples)";
 
+  inputs.cryptle.url = github:rvolosatovs/cryptle/feat/build;
```

Comment:
Why rvolosatovs/cryptle instead of enarx/cryptle?

Reply:
See commit message

Reply:
And https://github.com/enarx/codex/pull/6#issuecomment-1191620848